package.json

```diff
@@ -20,12 +20,12 @@
   },
   "dependencies": {
     "@slack/client": "^3.5.1",
-    "canvas": "^1.4.0",
-    "canvas-text-wrapper": "^0.9.2",
     "concat-stream": "^1.5.1",
     "cross-env": "^2.0.0",
     "dotenv": "^2.0.0",
+    "fs": "0.0.2",
     "lodash.assign": "^4.0.9",
```

Comment:
was there a reason to install `fs` from npm? there is a version of it on the registry, but it's a builtin.